<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ST-RA-WIN10-VM" failures="0" tests="2" name="tests.Firstphase" time="23.525" errors="1" timestamp="2021-12-02T14:23:39 GMT" skipped="0">
  <testcase classname="tests.Firstphase" name="Test15_AI0043" time="13.768"/>
  <system-out/>
  <testcase classname="tests.Firstphase" name="closeoff" time="9.757">
    <error message="For input string: &quot;[[FirefoxDriver: firefox on WINDOWS (eca6ae30-3478-469f-8cb0-5aaaac039347)] -&gt; xpath: (.//div[@id=&quot;conntable_info&quot;]/b)[1]]&quot;" type="java.lang.NumberFormatException">
      <![CDATA[java.lang.NumberFormatException: For input string: "[[FirefoxDriver: firefox on WINDOWS (eca6ae30-3478-469f-8cb0-5aaaac039347)] -> xpath: (.//div[@id="conntable_info"]/b)[1]]"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:668)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at pages.ConnectionPage.TableContent(ConnectionPage.java:125)
at pages.ConnectionPage.DeleteConnection(ConnectionPage.java:193)
at tests.TestBase.closeoff(TestBase.java:455)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- closeoff -->
  <system-out/>
</testsuite> <!-- tests.Firstphase -->
